sdk/pinocchio/src/sysvars/epoch_rewards.rs

```diff
@@ -0,0 +1,50 @@
+use super::Sysvar;
+use crate::impl_sysvar_get;
+
+pub const HASH_BYTES: usize = 32;
+
+#[derive(Default, Debug, Clone)]
+pub struct Hash([u8; HASH_BYTES]);
+
+impl Hash {
+    pub fn to_bytes(self) -> [u8; HASH_BYTES] {
+        self.0
+    }
+}
+
+#[repr(C, align(16))]
```

Comment:
*minor*

This `align` attribute is redundant:
> The alignment of the struct is the alignment of the most-aligned field in it.

https://doc.rust-lang.org/reference/type-layout.html#r-layout.repr.c.struct.align

Reply:
Still is redundant.  I do not have a strong opinion either way.
I see no reason to have it, so I would lean towards removing it.
But it is unlikely that the content of the struct changes to make this alignment overly aggressive, right?

Reply:
Agree, let's remove the unnecessary `align(16)` attribute.